src/components/browser-storage.ts

```diff
@@ -0,0 +1,19 @@
+const KEY = "redux";
```

Comment:
Lets make a habit to to use separate config file to import CONSTANTS 

/src/config/appConfig.js may be